fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -288,6 +288,10 @@ private void record(StoreTimer.Event event, long durationNanos) {
             if (name.endsWith(".cfs") || name.endsWith(".si") || name.endsWith(".cfe")) {
                 try {
                     readBlock(name, CompletableFuture.completedFuture(fileReference), 0);
+                    int lastBlock = (int) (fileReference.getSize() / fileReference.getBlockSize());
```

Comment:
Probably deserves a comment on why this is a worthwhile thing to do.

Reply:
:)

Reply:
Added Comment